### PR TITLE
Add initialize argument for create view instance.

### PR DESCRIPTION
**Pull request by peco2282**
## Summary

This commit allows the Item to be added when creating a new View instance.

Example:
```py
button1 = Button()
button2 = Button()
button3 = Button()

view = View(button1, button2, button3, timeout=50)
```

## Checklist

- [x] If code changes were made then they have been tested.
    - [x] I have updated the documentation to reflect the changes.
- [ ] This PR fixes an issue.
- [ ] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [ ] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

File: discord/ui/view.py [hunk 1]
```diff
@@ -178,7 +178,7 @@ def _init_children(self) -> List[Item[Self]]:
             children.append(item)
         return children
 
-    def __init__(self, *, timeout: Optional[float] = 180.0):
+    def __init__(self, *items: Item, timeout: Optional[float] = 180.0):
```
Comment (Puncher1) [inline discord/ui/view.py:181]:
```suggestion
    def __init__(self, *items: Optional[Item], timeout: Optional[float] = 180.0):
```
Also edit the docs at line 150:
```
    *items: Optional[:class:`Item`]
        The items to add to the view.

```

**Review by Puncher1 — APPROVED** [review]

File: discord/ui/view.py (hunk 1 above)
Comment (bijij) [reply discord/ui/view.py:181]: This doesn't look like the right use of `Optional`.
It should however be annotated as `Item[Any]`

Comment (Puncher1) [reply discord/ui/view.py:181]: Updated

**Comment by Rapptz** [comment]
Sorry, I deliberately don't want to hold all variadic arguments for something like this and is unlike anything in the library. I also don't think this is common enough to warrant the change. Thanks though!

**Pull request closed without merging**